cms/static/cms/js/require-config.js

```diff
@@ -27,7 +27,8 @@
             "mustache": "js/vendor/mustache",
             "codemirror": "js/vendor/codemirror-compressed",
             "codemirror/stex": "js/vendor/CodeMirror/stex",
-            "jquery": "js/vendor/jquery.min",
+            "jquery": "common/js/vendor/jquery",
+            "jquery-migrate": "common/js/vendor/jquery-migrate",
             "jquery.ui": "js/vendor/jquery-ui.min",
```

Comment:
As jQuery Migrate is loaded before any of the other jQuery plugins I think it should do fine at making sure all plugins retain compatibility. Longer-term I would like to take a look at upgrading/NPM-izing/potentially removing many of these though.